### DISCUSSION TITLE
Reporting gives an error

### DISCUSSION THREAD

**Discussion by radfaraf** [civitai/civitai#106] [General]
I tried reporting a comment because it seems to be referring to illegal stuff. Did both TOS violation and needs moderator review as the type of reporting on two separate message. It seems to submit and then instantly give an error message that it can't send. So not sure if anyone is getting these.

**Comment by JustMaier** [comment]
Sorry about that. We just rebuilt the reporting system today and are still working out the kinks. We’ll dig into that tomorrow and let you know when it’s fixed 👍

**Comment by JustMaier** [comment]
@radfaraf this has been fixed. Thanks again for reporting this!
